third_party/java/java_tools/update_java_tools.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
```

Comment:
nit: how about adding `-eu`: `-e` makes the shell exit early if a command returns a non-zero exit code and `-u` makes using undeclared variables an error (one less opportunity for typos!)

If you want to be fancy, you can also add `-x` that makes bash print every command it executes.

Reply:
I didn't add any of those because:
- `-e` because I still want to update some of the tools even if the others didn't build. I find it easier to debug if we keep the errors and print them at the end

- `-u` because I get an error of `$1` not being declared when I check if it equals "help", even though I do this check only after checking that $1 is not empty

- `x` because bazel already pollutes the output a lot, by looking at it is more than enough

Let me know if you think otherwise.

Reply:
On a second thought, I added `-e`. It can be more helpful to exit at the first error and not scroll through the whole log. :laughing: